CHANGES.md

```diff
@@ -2,7 +2,6 @@ Release Notes
 =============
 
 
-
```

Comment:
Unsure why this file was edited however it doesn't seem like an important change

Reply:
This was fixed in the latest push.